packages/esm-ohri-pmtct-app/src/config-schema.ts

```diff
@@ -18,9 +18,87 @@ export const configSchema = {
+      artInitiationConcept: '6e62bf7e-2107-4d09-b485-6e60cbbb2d08',
+      birthCountConcept: '1568AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      visitDate: '163260AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      hivTestStatus: '164401AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      artLinkage: '6e62bf7e-2107-4d09-b485-6e60cbbb2d08',
+      recenctViralLoad: '163310AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      motherStatusConcept: '1856AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      ancVisitConcept: '160446AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      infantDeliveryGroupingConcept: '1c70c490-cafa-4c95-9fdd-a30b62bb78b8',
+      infantPTrackerIdConcept: '164803AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      infantDateOfBirth: '164802AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+
+      //Infant Postnatal
+      infantPostnatalEncounterType: 'af1f1b24-d2e8-4282-b308-0bf79b365584',
+      nextVisitDate: '5096AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      ChildPTrackerId: '6c45421e-2566-47cb-bbb3-07586fffbfe2',
+      childDateOfBirth: '163260AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      artProphylaxisStatus: '1148AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      linkedToArt: 'a40d8bc4-56b8-4f28-a1dd-412da5cf20ed',
+      breastfeedingStatus: '1151AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      outcomeStatus: '160433AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      infantVisitDate: '159599AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      finalTestResults: '164460AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+
+      //Antenatal
+      antenatalEncounterType: '2549af50-75c8-4aeb-87ca-4bb2cef6c69a',
+      visitDateConcept: '163260AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      hivTestResultConcept: '159427AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      artNoConcept: '164402AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      followUpDateConcept: '5096AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      eDDConcept: '5596AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      nextVisitDateConcept: '5096AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      artStartDate: '159599AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      ancVisitsConcept: '1425AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      testTypeConcept: 'ee8c0292-47f8-4c01-8b60-8ba13a560e1a',
+      infantExposureStatus: '6027869c-5d7e-4a82-b22f-6d9c57d61a4d',
+
+      //Mother Postnatal
+      motherPostnatalEncounterType: '269bcc7f-04f8-4ddc-883d-7a3a0d569aad',
+      MothervisitDate: '163260AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      MotherHivStatus: '159427AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      MotherViralLoadDate: '163281AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      MotherViralLoadResult: '1305AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      MotherNextVisitDate: '5096AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      artUniqueNoConcept: '164402AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+
+      //Reports
+      ancVisitsReport: '5299521a-7fad-47bb-8280-14c99d04c790',
+      eddReport: 'aac635c9-0a77-4489-8b0d-866b3ad22f73',
+      motherHivStatusReport: 'ed50a889-dd5b-4759-861c-b54e3c686fe7',
+
+      //MCH Summary
+      mchEncounterType: '12de5bc5-352e-4faf-9961-a2125085a75c',
```

Comment:
@pirupius , do we need these comments or should I drop them?

Reply:
@lucyjemutai no we don't need the comments also encounter types are supposed to be in `encounterTypes` key and shouldn't be mixed with concepts. Please refer to the ticket description. I added this link for reference https://github.com/UCSF-IGHS/openmrs-esm-ohri/blob/c72a2e251ef8416ab7460b49ea2a7c564983c28b/packages/esm-tb-app/src/config-schema.ts#L11